File: packages/sdk-java/src/main/java/com/zenvio/sdk/model/SmsCancelResponse.java
package com.zenvio.sdk.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SmsCancelResponse {
    private boolean success;
    private Data data;

    public boolean isSuccess() { return success; }
    public void setSuccess(boolean success) { this.success = success; }
    public Data getData() { return data; }
    public void setData(Data data) { this.data = data; }

    @JsonIgnoreProperties(ignoreUnknown = true)
    public static class Data {
        private String smsId;
        private String status;

        @com.fasterxml.jackson.annotation.JsonProperty("sms_id")
        public String getSmsId() { return smsId; }
        @com.fasterxml.jackson.annotation.JsonProperty("sms_id")
        public void setSmsId(String smsId) { this.smsId = smsId; }
        public String getStatus() { return status; }
        public void setStatus(String status) { this.status = status; }
    }
}

File: packages/sdk-dotnet/src/Zenvio/Models/Sms/SmsCancelResponse.cs
using System.Text.Json.Serialization;

namespace Zenvio.Models.Sms;

public record SmsCancelResponse(
    [property: JsonPropertyName("success")] bool Success,
    [property: JsonPropertyName("data")] SmsCancelData Data
);

public record SmsCancelData(
    [property: JsonPropertyName("sms_id")] string SmsId,
    [property: JsonPropertyName("status")] string Status
);
